kafka-connect/src/test/java/com/hazelcast/jet/contrib/connect/KafkaConnectRabbitMQIntegrationTest.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.contrib.connect;
+
+import com.hazelcast.jet.Job;
+import com.hazelcast.jet.config.JobConfig;
+import com.hazelcast.jet.core.JetTestSupport;
+import com.hazelcast.jet.pipeline.Pipeline;
+import com.hazelcast.jet.pipeline.test.AssertionCompletedException;
+import com.hazelcast.jet.pipeline.test.AssertionSinks;
+import com.rabbitmq.client.AMQP;
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.ConnectionFactory;
+import org.apache.kafka.connect.data.Values;
+import org.junit.Rule;
+import org.junit.Test;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.RabbitMQContainer;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Objects;
+import java.util.Properties;
+import java.util.concurrent.CompletionException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+public class KafkaConnectRabbitMQIntegrationTest extends JetTestSupport {
+
+    public static final int ITEM_COUNT = 10_000;
+
+    @Rule
+    public RabbitMQContainer rabbitMQContainer = new RabbitMQContainer()
+            .withExchange("ex", "direct")
+            .withQueue("test-queue")
+            .withNetwork(Network.newNetwork());
+
+    @Test
+    public void readFromRabbitMqViaConnect() throws Exception {
```

Comment:
looks like in this test we first publish messages then start the job, how about adding another test or extending this one so that some messages are published after we've started the job ?